alpha/apps/kaltura/modules/extwidget/actions/embedIframeAction.class.php

```diff
@@ -122,18 +122,18 @@ public function execute()
 			$uiconf_id = $widget->getUiConfId();
 		if(!$uiconf_id)
 			KExternalErrors::dieError(KExternalErrors::MISSING_PARAMETER, 'uiconf_id');
-			
+
 		$partner_host = myPartnerUtils::getHost($partner_id);
 		$partner_cdnHost = myPartnerUtils::getCdnHost($partner_id);
 
 		$uiConf = uiConfPeer::retrieveByPK($uiconf_id);
 		if(!$uiConf)
 			KExternalErrors::dieError(KExternalErrors::UI_CONF_NOT_FOUND);
-			
+
 		$partner_host = myPartnerUtils::getHost($partner_id);
 		$partner_cdnHost = myPartnerUtils::getCdnHost($partner_id);
-		
-		$html5_version = kConf::get('html5_version');
+
+		$html5_version = kConf::getArrayValue('html5_version','playerApps',kConfMapNames::APP_VERSIONS,null);
```

Comment:
on all places you call to getArrayValue add spaces after each ','

Reply:
@inbal-ben-david may I suggest using a formatting tool that will automatically format the code according to standards
https://github.com/prettier/plugin-php - highly recommended!